config/settings.yml

```diff
@@ -28,6 +28,11 @@ was_crawl_dissemination:
 
   path_working_directory: "/web-archiving-stacks/data/indices/path_working/"
   main_path_index_file: "/web-archiving-stacks/data/indices/path/path-index.txt"
+  # when in production with pywb, new settings for path:
+  # path_working_directory: "/web-archiving-stacks/data/indexes/path_working/" 
+  # main_path_index_file: "/web-archiving-stacks/data/indexes/path/path-index.txt"
+  # Needed only while pywb is run in parallel with openwayback:
+  new_main_path_index_file: "/web-archiving-stacks/data/indexes/path/path-index.txt"
```

Comment:
can we call this `pywb_main_path_Index_file`?  Things that are "new" have a way of getting "old" and we don't always have the time to fix up the code to reflect that.

Reply:
That makes sense, updating.